ydb/core/tx/schemeshard/schemeshard_import_flow_proposals.cpp

```diff
@@ -226,5 +226,6 @@ THolder<TEvIndexBuilder::TEvCancelRequest> CancelIndexBuildPropose(
     return MakeHolder<TEvIndexBuilder::TEvCancelRequest>(ui64(indexBuildId), domainPath.PathString(), ui64(indexBuildId));
 }
 
+
```

Comment:
Since you still have to do another commit for the tests to compile. Could you remove the extra newline here?

Reply:
yes